wellcomeml/__version__.py

```diff
@@ -1,5 +1,5 @@
 __name__='wellcomeml'
-__version__='2020.2.1'
+__version__='2020.2.2'
```

Comment:
Maybe don't increment the version just yet until we are ready for a new release.